### PR TITLE
Move migration4.md to correct folder & fix merge errors

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: src/GraphQL.sln [hunk 1]
```diff
@@ -112,6 +109,15 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ISSUE_TEMPLATE", "ISSUE_TEM
 		..\.github\ISSUE_TEMPLATE\question.md = ..\.github\ISSUE_TEMPLATE\question.md
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "migrations", "migrations", "{3C62C155-7603-4926-A4A5-C91AED6C4EA7}"
+	ProjectSection(SolutionItems) = preProject
+		..\docs2\site\docs\guides\migration.md = ..\docs2\site\docs\guides\migration.md
+		..\docs2\site\docs\guides\migration3.md = ..\docs2\site\docs\guides\migration3.md
```
Comment (Shane32) [inline src/GraphQL.sln:115]:
```suggestion
		..\docs2\site\docs\migrations\migration.md = ..\docs2\site\docs\migrations\migration.md
		..\docs2\site\docs\migrations\migration3.md = ..\docs2\site\docs\migrations\migration3.md
```

File: src/GraphQL.sln [hunk 2]
```diff
@@ -112,6 +109,15 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ISSUE_TEMPLATE", "ISSUE_TEM
 		..\.github\ISSUE_TEMPLATE\question.md = ..\.github\ISSUE_TEMPLATE\question.md
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "migrations", "migrations", "{3C62C155-7603-4926-A4A5-C91AED6C4EA7}"
+	ProjectSection(SolutionItems) = preProject
+		..\docs2\site\docs\migrations\migration.md = ..\docs2\site\docs\migrations\migration.md
```
Comment (Shane32) [inline src/GraphQL.sln:114]:
```suggestion
		..\docs2\site\docs\migrations\migration2.md = ..\docs2\site\docs\migrations\migration2.md
```

**Comment by Shane32** [comment]
@sungam3r I fixed the documentation problem, but the code won't compile on Windows.  Some test runs seem to work fine while for others it can't find Shouldly.  Perhaps the .Net Framework 4.8 is not compatible with Shouldly 4.0.0-beta0004.  In any case, I'm not to sure what to try next.  I did try reverting to Shouldly 0002 but that introduces a different bug.  All tests (with all 3 TFMs) work fine locally.

**Comment by sungam3r** [comment]
Ah! Migration4 and other files were moved, of course 🤦 . I will check failed tests.

**Review by sungam3r — APPROVED** [review]

**Comment by sungam3r** [comment]
Give me some time to modify this PR.

**Comment by Shane32** [comment]
Thanks! Have at it!

**Comment by sungam3r** [comment]
Two options : 
1. Just drop `net48` TFM for tests.
2. Play with `sn -Vr ` command for `net48` TFM.

Perhaps for the Shouldly 4.0.0 (stable) we can return `net48`. I don't know why this error occurs, possibly due to delay signed option.

**Comment by sungam3r** [comment]
see https://github.com/shouldly/shouldly/issues/707

**Comment by sungam3r** [comment]
@Shane32 I have no .NET Framework installed, how about you ? `C:\Program Files\Microsoft SDKs\Windows\v6.0A\bin\sn.exe` can check if assembly is properly signed.

**Comment by Shane32** [comment]
I'll check it out. 

I agree; sounds like dropping the net48 TFM for now is the most practical option. Thanks!

**Comment by Shane32** [comment]
@sungam3r Shouldly beta 5 is working with the net48 TFM; I'll merge these changes into develop.

**Pull request merged**